optuna_integration/comet/comet.py

```diff
@@ -213,6 +216,7 @@ def _init_optuna_trial_experiment(
         setattr(comet_ml, "active_experiment", experiment)
         return experiment
 
+    @experimental_func("4.0.0")
```

Comment:
Since the entire class is experimental, isn't a decorator unnecessary for the method?

Reply:
Thanks. You're right. I just followed similar implementations, such as https://github.com/optuna/optuna-integration/blob/605390c85f86f7eef621e6339dd9fcd41db26611/optuna_integration/wandb/wandb.py#L187, but this looks unnecessary too.